### DISCUSSION TITLE
unsigned.io unreachable?

### DISCUSSION THREAD

**Discussion by cobraPA** [markqvist/Reticulum#296] [Help & Questions] [answered]
Hi @markqvist, looks like unsigned.io is not responding here on the USA East coast at the moment...  Just a heads up in case you aren't updating the site or something.

**Comment by markqvist** [comment, marked as answer]
Thanks a lot for the heads up! I was just doing some updates on the server, and it should be back up now :)

**Reply by cobraPA** [reply]
Yes, all back to normal from here.
